alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultProviderDataAccessor.java

```diff
@@ -85,6 +85,7 @@ public DefaultProviderDataAccessor(ProviderProjectRepository providerProjectRepo
                            .collect(Collectors.toList());
             }
         } catch (AlertDatabaseConstraintException ignored) {
+            // ignoring this exception
```

Comment:
SonarCloud does not like an empty catch block unless there is a comment as to why

Reply:
I think we as a team agreed to the convention of naming the exception `ignored`. We should remove this SonarCloud rule.

Reply:
Is there a way for us to handle this type of Exception differently? It's all over the code base and almost always has the same content. Would handling these exceptions at the source be a better option?